src/main/java/constants/Constants.java

```diff
@@ -0,0 +1,7 @@
+package constants;
+
+public class Constants {
+    public static final String VALUE_DRIVER_SET_PROPERTY = "webdriver.chrome.driver";
+    public static final String PATH_SET_PROPERTY = "/Users/ilsemacias/workspace/drivers/chromedriver";
```

Comment:
This should be read from an initial configuration, not a static constant if this want to be shared